src/Controller/Admin/ImportAssetsController.php

```diff
@@ -35,7 +38,12 @@ public function __construct(
     )]
     public function import(Request $request): JsonResponse
     {
-        return parent::import($request);
+        $response = parent::import($request);
+        foreach ($this->cleanUpFiles as $file) {
+            @unlink($file);
+        }
+
+        return $response;
```

Comment:
How about this?

```suggestion
        try {
            return parent::import($request);
        } finally {
            foreach ($this->cleanUpFiles as $file) {
                @unlink($file);
            }
        }
```

It should also clean up partial imports that fail somewhere in between, shouldn't it?

Reply:
Wow...this works though the return statement..crazy, never seen before and not should if I would have understood what happens if had seen 😂🙈

Reply:
Yes, this is how finally works. It is executed *after* the return.